inetum-realjobs-frontend/src/app/components/user-details/user-details.component.html

```diff
@@ -5,6 +5,15 @@
 </div>
 
 <div class="p-fluid" *ngIf="(!loading && !error)">
+
+  <div class="wrapper">
+    <div>
+      <h4 class="text_manageAccount">Manage Account</h4>
+    </div>
+    <div>
+      <img class="profile_pic" src="https://www.w3schools.com/howto/img_avatar.png" alt="Avatar">
+    </div>
+  </div>
```

Comment:
This is not connected to the fields, shouldn't these be in there? Maybe look at cards from primeng